### ISSUE TITLE
addSiblings() issus

### ISSUE THREAD

**Issue by jonasgrundtvig** [dabeng/OrgChart#254]
I have a problem when I'm adding a sibling to a node, then its parent get a wrong colspan and the new node is not appended correctly.

The result looks like this:
![skaermbillede 2017-07-28 kl 11 57 39](https://user-images.githubusercontent.com/9499676/28712726-829fa270-738c-11e7-80de-0bac36279a3d.png)
Child 4 is the new sibling added to Child 1

Init chart:
```javascript
initChart: function (data) {
        var self = this;

        self.orgChart = self.ui.orgChart.orgchart({ 
            data: { id: 'null', name: "Department",  children: data.toJSON() },
            draggable: true,
            pan: true,
            zoom: true,
            createNode: function($node, data) {
                $node[0].id = data.id;
            }
        });

        self.orgChart.$chart
            .on('click', '.node', function() {
              self.node = $(this);
            });

        self.orgChart.$chart.addClass('noncollapsable');
        console.log('chart: ', self.orgChart);
        this.moveNode();
    },
```

Creating the new node:
```javascript
var data = [{ 
    'name': model.get('name'), 
    'id': model.get('id')
}];

switch (type) {
    case 'sibling':
        data.relationship = "110";
        self.orgChart.addSiblings($node, { 'siblings': data });
        console.log("addSiblings", data);
        break;
    default:
        var hasChild = $node.parent().attr('colspan') > 0 ? true : false;
        if (!hasChild) {
            data.relationship = "100";
            self.orgChart.addChildren( $node, { 'children': data } );
            console.log("addChildren", data);
        } else {
            data.relationship = "110";
            var node = $node.closest('tr').siblings('.nodes').find('.node:first');
            self.orgChart.addSiblings(node, { 'siblings': data });
            console.log("addChildren - addSiblings", data);
        }
}

resolve();
```

Thank you for a great plugin and for your help.

**Comment by jonasgrundtvig** [comment]
FIX: jQuery updated from 2.2.3 to 3.2.1

**Comment by dabeng** [comment]
Thanks a lot.
